src/fs/driver/initfs/initfs_dvfs.c

```diff
@@ -246,5 +246,5 @@ static struct dumb_fs_driver initfs_dumb_driver = {
 	.mount_end = initfs_mount_end,
 };
```

Comment:
Ok, May be you are right. I did it.
But 'dvfs' needs deep reworking and I don't want to make it in this branch.